kratos/tests/test_eigen_solvers.py

```diff
@@ -188,6 +188,8 @@ def test_FEAST_with_mkl_solver(self):
             self.skipTest("KratosMultiphysics.ExternalSolversApplication is not available")
         try:
             import KratosMultiphysics.EigenSolversApplication
+            if not hasattr(KratosMultiphysics.EigenSolversApplication, "ComplexPardisoLUSolver"):

```

Comment:
i think you should add this prior to line 190

Reply:
No this is ok, the EigenSolversApplication availability is checked in the lines above.
This is just for the case that EigenSolversApplication is compiled without MKL.